docs/install/containers/recipes/index.md

```diff
@@ -144,9 +147,20 @@ docker run -e ZEO_ADDRESS=zeo:8100 --link zeo plone/plone-backend pack
 ```
 
 In RelStorage mode, pass the connection DSN.
+The following command assumes that the service that runs the Plone instance is named `backend`.
+Replace `backend` with your container's name.
+
+```shell
+docker run -e RELSTORAGE_DSN="dbname='plone' user='plone' host='db' password='password' port='5432'" backend pack
```

Comment:
```suggestion
docker run -e RELSTORAGE_DSN="dbname='plone' user='plone' host='db' password='password' port='5432'" plone/plone-backend pack
```

Sorry @stevepiercy I mixed things when reviewing: with `docker run ` we need to specify the docker image name, and with `docker exec` the running container name.

Reply:
Yes, sorry.

Reply:
Done in https://github.com/plone/documentation/pull/1899/commits/46b5d64d52035bbb2fa9275458ac7b3424d5d70d